### ISSUE TITLE
Specifying an operation name when the query contains only one operation generates a spurious error

### ISSUE THREAD

**Issue by hlship** [walmartlabs/lacinia#121]
Unfortunately, this turned up in the lacinia-pedestal test suite, and was obscured by the (old) way of releasing RCs internally.

The error was introduced in 0.22.0 (not present when lacinia-pedestal depended on 0.21.0).

```
query stuff($v: String) {
                                      echo(value: $v) { value }
                            }
```

results in:

```
{:body {:errors [{:message "Single operation did not provide a matching name."
                  :op-name "stuff"}]}
 :status 400}
```

**Comment by hlship** [comment]
Perhaps this is a bug in lacinia-pedestal. Can't reproduce so far in Lacinia.

**Comment by hlship** [comment]
Ah, found the problem.  Need to use the full 5 argument version of lacinia/execute (to specify the options).

**Comment by hlship** [comment]
Fixed in #122
